java-agent/src/nativeMain/kotlin/com/epam/drill/agent/configuration/AgentParameters.kt

```diff
@@ -29,6 +29,6 @@ data class AgentParameters(
     val isKafka: Boolean = false,
     val isCadence: Boolean = false,
     val isTlsApp: Boolean = false,
-    val coverageRetentionLimit: Long  = 2000,
+    val coverageRetentionLimit: String  = "200Mb",
```

Comment:
Reason for defining default in multiple places.
1. There is already a default initialization in `AgentArguments.kt`
2. Decide a single place to assign defaults. Preferably in `AgentArguments.kt` for the sake of consistency

Reply:
Edit: just set the same value `String  = "200Mb",`, leave refactoring AgentConfig/AgentArguments/AgentParams for a separate task

Reply:
done